source/text.cpp

```diff
@@ -547,7 +547,8 @@ spv_result_t spvTextEncodeOpcode(const spvtools::AssemblyGrammar& grammar,
     // The <opcode> after the '=' sign.
     context->setPosition(nextPosition);
     if (context->advance())
-      return context->diagnostic() << "Expected opcode, found end of stream.";
+      return context->diagnostic()
+             << opcodeName << " Expected opcode, found end of stream.";
```

Comment:
Why would you want an opcode name here?  The problem is that you reached the end of the stream and cannot read the opcode name.

Reply:
I guess maybe what I want it to say is `"Expected opcode, found end of stream after " << opcodeName`

When dealing with pipeline of tools and something in the middle breaks, it would be nice to have some extra information

Reply:
looked again, you are right, there is no need for the `opcodeName` here, reverted line